src/ui/components/DevTools.tsx

```diff
@@ -102,15 +95,6 @@ function DevTools({
   }
 
   if (!loaderResult) {
-    if (deleted_at) {
-      expectedError = { message: "This replay has been deleted." };
-    }
-
-    // Test recordings don't have a user, so we skip this check in that case.
-    if (user && user.invited == false) {
-      expectedError = { message: "The author of this Replay has not activated their account yet." };
```

Comment:
If I understand this correctly, it is actually a bug: it assumes that `invited` will be `true` for every user that has logged in at least once ("activated their account"), but that doesn't make any sense because users are only created when they login for the first time. Instead, `invited` should be `false` for users that logged in without receiving an invitation, but I don't think we want to block those users' recordings.
This bug was never noticed because it was neutralized by another bug that set `invited` to `true` even for users that didn't receive an invitation. That bug actually also neutralized another bug: instead of looking at invitations that the user has received, the code for setting `invited` looks at invitations that the user has sent...
So no, I don't think we need `recording.ownerNeedsInvite` (at least not for now).

Reply:
>  it assumes that invited will be true for every user that has logged in at least once ("activated their account"), but that doesn't make any sense because users are only created when they login for the first time. Instead, invited should be false for users that logged in without receiving an invitation, but I don't think we want to block those users' recordings.

My understanding is that, if a user logs in and has not been invited, their account would have `invited: false` until someone sends them an invitation, at which point their account will have `invited: true` set. The goal being that if you log in, you should be able to comment, but if you make a recording, you're not able to view it until you've been invited?

I'm fine to remove that logic, but should that be a bigger conversation?

Reply:
I have reinstated the old logic around invitations, which will mark any user as invited as soon as they login.
We should definitely clarify what's the intention here.